packages/shared/components/inputs/Dropzone.svelte

```diff
@@ -56,7 +56,7 @@
             />
             <Text type="h4">{localize('actions.dragDrop')}</Text>
             <Text classes="mb-12" type="p" secondary smaller>{extentionsLabel}</Text>
-            <Button secondary onClick={onDrop}>{localize('actions.chooseFile')}</Button>
+            <Button outline onClick={onDrop}>{localize('actions.chooseFile')}</Button>
```

Comment:
This button is different from what we have in the latest beta, but I guess is not super important. 
This PR 
![image](https://user-images.githubusercontent.com/3624944/190413852-b05b420a-6119-49c9-864d-3ba61692fda5.png)
Beta 
![image](https://user-images.githubusercontent.com/3624944/190413899-9b83350b-731a-43bf-a5b2-e52ec6407ebd.png)

Reply:
It is really different but I think we need to reach in an agreement of how the buttons should be, this PR uses the UI Kit's Button as reference